lib/screen/bloc/feralfile/feralfile_bloc.dart

```diff
@@ -28,8 +28,8 @@ class FeralfileBloc extends AuBloc<FeralFileEvent, FeralFileState> {
   Future<Persona?> getPersonaFromETHAddress(String address) async {
     final personas = await _cloudDB.personaDao.getPersonas();
     for (var persona in personas) {
-      final ethAddress = await persona.wallet().getETHEip55Address();
-      if (ethAddress == address) {
+      final ethAddresses = await persona.getEthAddresses();
```

Comment:
I missed something from the last pull request, can you please check and replace all the `wallet.getAddres(index)` to `wallet.getETHEip55Address(index)` in the app?